### PR TITLE
Make service wait time meaningful

### PR DESCRIPTION
**Pull request by fenglixa**
Make service wait time and interval meaningful
-Fix for #4585

### CODE REVIEW

**Comment by RA489** [comment]
@minikube-bot OK to test

**Comment by fenglixa** [comment]
Append my UT result:
image

**Comment by medyagh** [comment]
Do you mind adding a PR description how you made it better?

File: pkg/minikube/service/service.go [hunk 1]
```diff
@@ -245,7 +245,12 @@ func OptionallyHTTPSFormattedURLString(bareURLString string, https bool) (string
 // WaitAndMaybeOpenService waits for a service, and opens it when running
 func WaitAndMaybeOpenService(api libmachine.API, namespace string, service string, urlTemplate *template.Template, urlMode bool, https bool,
 	wait int, interval int) error {
-	if err := util.RetryAfter(wait, func() error { return CheckService(namespace, service) }, time.Duration(interval)*time.Second); err != nil {
+	// Convert "Amount of time to wait" and "interval of each check" to attempts
+	if interval == 0 {
+		interval = 1
+	}
+	attempts := wait/interval + 1
```
Comment (medyagh) [inline pkg/minikube/service/service.go]: Could you add a comment here why

**Comment by medyagh** [comment]
Looks good to me thank you for contribution :) this is gonna make user much happier

**Pull request merged**